src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java

```diff
@@ -191,24 +229,23 @@ public static void logImage(String command, File image, boolean success) {
     }
     logImage(command, new String(bytes, StandardCharsets.UTF_8), success);
   }
-
+  //TODO: Check error class here
   public static void logImage(String command, String imageAsBase64, boolean success) {
-    imageAsBase64 = "<img src=\"data:image/png;base64," + imageAsBase64 + "\">";
-    String className = success ? "success" : "error";
-    CommonUtils.appendTextToFile(logPath, ("<tr class=\"" + className + "\"><td>" + command
-        + "</td><td>" + imageAsBase64 + "</td><td> <br/> &nbsp;</td></tr>"));
+    String imgHtml = VelocityWrapper.appendImage(imageAsBase64);
+    String className = success ? SUCCESS_CLASS : ERROR_CLASS;
+    String html = VelocityWrapper.appendLogRow(Arrays.asList(className), "INFO", imgHtml);
+    CommonUtils.appendTextToFile(logPath, html);
   }
 
   private static void logJSError(WebDriver driver) {
-    if ("true".equals(Configuration.getJSErrorsEnabled())) {
+    if (String.valueOf(true).equals(Configuration.getJSErrorsEnabled())) {
       JavascriptExecutor js = (JavascriptExecutor) driver;
       List<String> error =
           (ArrayList<String>) js.executeScript("return window.JSErrorCollector_errors.pump()");
       if (!error.isEmpty()) {
-        StringBuilder builder = new StringBuilder();
-        builder.append(
-            "<tr class=\"error\"><td>click</td><td>" + error + "</td><td> <br/> &nbsp;</td></tr>");
-        CommonUtils.appendTextToFile(logPath, builder.toString());
+        //TODO:Check error to avoid toString method
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1135)